packages/docker-node/src/errors.ts

```diff
@@ -0,0 +1,67 @@
+import { Image } from './types'
+
+export class DockerError extends Error {
+  constructor(public readonly parent?: Error) {
+    super()
```

Comment:
Why only this error is instantiated with `super()` while others are instantiated with `super(parent)`